config.m4

```diff
@@ -10,7 +10,6 @@ PHP_ARG_ENABLE(simdjson, whether to enable simdjson, [ --enable-simdjson   Enabl
 
 if test "$PHP_SIMDJSON" != "no"; then
 
-  CXXFLAGS=" -march=native -O2 -std=c++17 "
```

Comment:
Drop this to honor system build flag
Can de add in PHP_NEW_EXTENSION as "additional" flags